src/write_prometheus.c

```diff
@@ -167,7 +232,23 @@ static char *format_labels(char *buffer, size_t buffer_size,
              escape_label_value(value, sizeof(value), m->label[i]->value));
   }
 
-  strjoin(buffer, buffer_size, labels, m->n_label, ",");
+  /* Add custom labels */
```

Comment:
I think it has no much sense to build array of custom label values each time when `format_labels()` called.
It would be better to build string of custom labels once, preferrable in `prom_config()`, and then just append it in `format_labels()`. Also, no `calloc()/free()` will be introduced here. 

What do you think about such improvement?